index.html

```diff
@@ -0,0 +1,232 @@
+          <div class="general">
+              <h3>Tonic</h3>
+              <div id="general_inf">
+                 <p class="title">CANOPY</p>
+                 <img class="counter" src="images/Counter.png" alt="counter"/>
+                 <p class="job-title">Back End Dev</p>
+                 <img class="counter" src="images/Counter.png" alt="counter">
+                 <p class="year">2015</p>
+              </div>
+              <p class="project_description">
+                      A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <footer>
+                  <ul id="skills_0" class="skills">
+                     <li>html</li>
+                     <li>css</li>
+                     <li>javaScript</li>
+                  </ul>
+                  <div class="external_link">
+                     <a href="#">See Project</a>
+                  </div>
+              </footer>
+            </div>
+        </section>
+        <section id="p2">
+          <img  class="project_image" src="images/Snapshoot_Portfolio1.png" alt="snapshoot image project">
+          <div class="general">
+            <h3>Multi-Post Stories</h3>
+            <div id="general_inf_1">
+              <p class="title">CANOPY</p>
+              <img class="counter" src="images/Counter.png" alt="counter"/>
+              <p class="job-title">Back End Dev</p>
+              <img class="counter" src="images/Counter.png" alt="counter">
+              <p class="year">2015</p>
+            </div>
+            <p class="project_description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            
+            <footer>
+               <ul id="skills_1" class="skills">
+                 <li>html</li>
+                 <li>css</li>
+                 <li>javaScript</li>
+               </ul>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+            </footer>
+          </div>
+
+        </section>
+        <section id="p3">
+            <img  class="project_image" src="images/Snapshoot_Portfolio2.png" alt="snapshoot image project">
+            <div class="general">
+                <h3>Tonic</h3>
+                <div id="general_inf_2">
+                  <p class="title">CANOPY</p>
+                  <img class="counter" src="images/Counter.png" alt="counter"/>
+                  <p class="job-title">Back End Dev</p>
+                  <img class="counter" src="images/Counter.png" alt="counter">
+                  <p class="year">2015</p>
+                </div>
+                <p class="project_description">
+                     A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <footer>
+                   <ul id="skills_2" class="skills">
+                    <li>html</li>
```

Comment:
- [ ] Nice work on implementing the work section of this milestone, but some issues were missed in your design when aligning the items; please, as much as possible, stick to the Figma template you choose. :smile_cat: 

| Figma design | Your design |
|-------------------|------------------|
|![Screenshot from 2022-11-02 12-59-12](https://user-images.githubusercontent.com/65129228/199473933-1daee9b8-00a8-46a7-b245-46a46eab4e6d.png)|![Screenshot from 2022-11-02 14-03-49](https://user-images.githubusercontent.com/65129228/199474045-373a6f08-8607-4094-bedd-79d8ae89ea79.png)|